checker/src/org/checkerframework/checker/genericeffects/GenericEffectVisitor.java

```diff
@@ -1,16 +1,12 @@
 package org.checkerframework.checker.genericeffects;
 
-import com.sun.source.tree.ClassTree;
-import com.sun.source.tree.MemberSelectTree;
-import com.sun.source.tree.MethodInvocationTree;
-import com.sun.source.tree.MethodTree;
-import com.sun.source.tree.NewClassTree;
+import com.sun.source.tree.*;
```

Comment:
Please do not use wildcard imports.  This is probably why the code hygiene CI job failed --- there's an automated check on every PR that certain aspects of good style are followed.  Avoiding wildcards is one of them.

Please replace this with the original series of imports of ```com.sun.source.tree``` classes, in their original order (to simplify the diff), plus anything new you might require that wasn't previously necessary.